hooks/post_gen_project.py

```diff
@@ -76,9 +76,6 @@ def move_web_client_to_root(client):
         rmtree("client")
     move(join(web_clients_path, client), join("client"))
     rmtree(join(web_clients_path))
-    env_path = join("client", ".env.local.example")
-    if exists(env_path):
-        rename(env_path, join("client", ".env.local"))
```

Comment:
The point of this example file is for all devs joining a project, not just the one who started it.
So don't rename it. We have docs that tell you to copy this file while getting set up.